### PR TITLE
BUG FIX : made isString() available to  upperCase() scope

### PR DESCRIPTION
**Pull request by mooSylla**
## Goal of the pull request:
* [ ] Documentation
* [X] Bugfix
* [ ] Feature (New!)
* [ ] Enhancement

## Description
#105

### CODE REVIEW

File: src/common-utils/index.js [hunk 1]
```diff
@@ -71,9 +75,7 @@ module.exports = {
     },
 
     // Returns if a value is a string
-    isString(value) {
-        return typeof value === 'string' || value instanceof String;
-    },
+    isString,
```
Comment (pajaydev) [inline src/common-utils/index.js:79]: I would suggest this fix recommended by @midrissi to bring the isString in scope

https://github.com/pajaydev/ebay-node-api/blob/master/src/common-utils/index.js#L68-L71

```
upperCase(data) {
  if (!this.isString(data)) data = data.toString();
  return data.toUpperCase();
},
```

Comment (mooSylla) [reply src/common-utils/index.js:79]: I tried it and it gave me ""this" is not defined""   there is actually no context "this" in that environment. 
However if you wanna bring the isString() in scope at all cost i'll see another alternative, i'm also open for your suggestions guys.

Comment (mooSylla) [reply src/common-utils/index.js:79]: Not working TypeError: Cannot read property 'isString' of undefined

Comment (pajaydev) [reply src/common-utils/index.js:79]: I think we need to bind the this while calling like this 
```
upperCase.call(this, marketPlaceId);
```

But I feel this is fine.

Comment (mooSylla) [reply src/common-utils/index.js:79]: `this` in that scope refers to the ebay object however i did this and it worked : 
`const Utils = require('./common-utils');`
`    marketPlaceId = upperCase.call(Utils, marketPlaceId);
`

Comment (mooSylla) [reply src/common-utils/index.js:79]: @pajaydev   Anyway thank you so much for mergeing my PR you really made my day :))))

Comment (pajaydev) [reply src/common-utils/index.js:79]: Cool 👍

**Pull request merged**